GitUI/ScriptsEngine/ScriptEvent.cs

```diff
@@ -15,6 +15,7 @@ public enum ScriptEvent
         BeforeMerge,
         AfterMerge,
         BeforeFetch,
-        AfterFetch
+        AfterFetch,
+        ShowInFileList
```

Comment:
This enum contains "git events". What does "ShowInFileList" mean in this context?

Reply:
`ShowInUserMenuBar` is also an event. This enum is called `ScriptEvent`, not `GitEvent`. It represents situations in which a script should appear/be executed.